es_gui/apps/data_manager/widgets.py

```diff
@@ -456,21 +456,348 @@ def execute_download(self):
 
             self.n_active_threads = len(job_batches)
 
+            five_minute_start = datetime.datetime(2017, 12, 1)
+            
             # (Re)set the progress bar and output log.
             self.progress_bar.value = 0
-            self.progress_bar.max = total_months*total_nodes + total_months
+            self.progress_bar.max = 2*total_months
             self.output_log.text = ''
 
             # Check connection settings.
             ssl_verify, proxy_settings = check_connection_settings()
 
             # Spawn a new thread for each download_ISONE_data call.
             for batch in job_batches:
-                thread_downloader = threading.Thread(target=self._download_ISONE_data, args=(acc_user, acc_pw, batch[0], batch[-1]),
+                thread_downloader = threading.Thread(target=self.download_ISONE_data_updated, args=(acc_user, acc_pw, batch[0], batch[-1]),
                                                      kwargs={'ssl_verify': ssl_verify, 'proxy_settings': proxy_settings, 'nodes':nodes_selected})
 
                 thread_downloader.start()
 
+    #//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////#
+    def download_ISONE_data_updated(self, username, password, datetime_start, datetime_end=None, nodes=[], typedat="all", path='data/', ssl_verify=True, proxy_settings={}):
+        """Downloads a month's worth of ISO-NE day ahead LMP and RCP data pre Dec 2017. Post Dec 2017 downloads da LMP, five minute LMP, and five minute RCP data.
+    
+        :param username: ISO-NE ISO Express username
+        :type username: str
+        :param password: ISO-NE ISO Express password
+        :type password: str
+        :param nodes: list of pricing nodes
+        :type nodes: list
+        :param datetime_start: the start of the range of data to download
+        :type datetime_start: datetime
+        :param datetime_end: the end of the range of data to download, defaults to one month's worth
+        :type datetime_end: datetime
+        :param path: root directory of data download location, defaults to os.path.join('data')
+        :param path: str, optional
+        :param ssl_verify: if SSL verification should be done, defaults to True
+        :param ssl_verify: bool, optional
+        """
+        print("Max attempts:" + str(MAX_WHILE_ATTEMPTS))
+        if not datetime_end:
+            datetime_end = datetime_start
+    
+        pathlistnodes = path
+        listnodes_file = os.path.join(pathlistnodes, '../es_gui/apps/data_manager/_static/', 'nodes_isone.csv')
+        #listnodes_file = 'C:/Users/wolis/Documents/QuESt/snl-quest-1.2.c/es_gui/apps/data_manager/_static/nodes_isone.csv'
```

Comment:
We should try to clean out commented code like this as well as comment out or remove superfluous print statements